.changeset/five-balloons-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@monite/sdk-react': patch
+---
+
+align pagination arrows center
```

Comment:
Fix it, please, according to the rules:

### Changeset messages (changelog)

1. **Content:** Use present simple for entries.
2. **TASK-ID:** End each entry with the task identifier (e.g., *(DEV-1234)*).
3. **Examples:**
    - `fix(DEV-3456): add social network authentication feature`

Reply:
fixed